style.css

```diff
@@ -1 +1,267 @@
-/*  Add your styling here */
+.main {
+    display: grid;
+    width: 100%;
+    height: auto;
+  
+  }
+  /*section one*/
+  .one{
+    background-color: white;
+    display: grid;
+    grid-template-rows: auto auto;
+  }
+  /*section one-one*/
+  .one-one {
+
+  
+  }
+  
+  .mobile-container {
+  
+    color: white;
+    border-radius: 10px;
+    justify-self: end;
+  }
+  
+  .topnav {
+    overflow: hidden;
+    position: relative;
+  }
+  
+  .topnav #myLinks {
+  display:hidden ;
```

Comment:
nice :)
display: none; works for me to show it closed first

Reply:
position: fixed;
right: 0px;
background-color: white;
z-index: 2;
would flow over the menu